flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/AggregateUtil.scala

```diff
@@ -785,7 +785,7 @@ object AggregateUtil {
     (propPos._1, propPos._2)
   }
 
-  private def transformToAggregateFunctions(
```

Comment:
agree 
@fhueske  - what do you think?
I also used the model from other such functions